### ISSUE TITLE
Keyboard shortcut help (accessible version) won't scroll

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#12362]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : RStudio 2022.11.0-daily+192 or any build since
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem

1. Start RStudio and turn on screen-reader support (Help / Accessibility / Screen Reader Support), let the app restart.
2. Help / Keyboard Shortcuts Help

### Describe the problem in detail

The plain HTML version of keyboard shortcuts help opens in web browser as it should, but it cannot be scrolled via keyboard or mouse.

This was introduced with this PR: https://github.com/rstudio/rstudio/pull/11966

### Describe the behavior you expected

Page should scroll. On the positive side, you can still read everything on this page with a screen reader since that doesn't rely on being able to scroll things into view...

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
Triage: Proposed for Elsbeth Geranium.

**Comment by ronblum** [comment]
Verified in RStudio Desktop 2022.12.0-preview+339 on MacOS 13.
